tests/CMakeLists.txt

```diff
@@ -1,7 +1,15 @@
 cmake_minimum_required(VERSION 3.10)
 
+set(COVERAGE OFF CACHE BOOL "Coverage")
+
 # Link runTests with what we want to test and the GTest and pthread library
 add_executable(runTests TestyMcTestFace.cpp)
-target_link_libraries(runTests usgscsm ${GTEST_LIBRARIES} ${GTEST_MAIN_LIBRARIES} pthread)
+target_link_libraries(runTests PUBLIC usgscsm ${GTEST_LIBRARIES} ${GTEST_MAIN_LIBRARIES} pthread)
+
+if (COVERAGE)
+  target_compile_options(runTests PRIVATE --coverage)
+  target_link_libraries(runTests PRIVATE --coverage)
+endif()
 
-gtest_discover_tests(runTests WORKING_DIRECTORY tests)
+gtest_discover_tests(runTests tests)
+message("Working dir: ${WORKING_DIRECTORY}")
```

Comment:
Why not set the working directory? How do we ensure data file paths for testing specifying working directory?

Reply:
I need to pull the extraneous message